package.json

```diff
@@ -72,29 +72,29 @@
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-2": "6.24.1",
     "coveralls": "2.13.1",
-    "eslint": "4.0.0",
-    "eslint-config-prettier": "2.1.1",
-    "eslint-plugin-flowtype": "2.34.0",
+    "eslint": "4.2.0",
+    "eslint-config-prettier": "2.3.0",
+    "eslint-plugin-flowtype": "2.34.1",
     "eslint-plugin-react": "7.1.0",
-    "flow-bin": "0.47.0",
-    "husky": "0.13.4",
-    "intl-messageformat": "1.3.0",
-    "intl-relativeformat": "1.3.0",
+    "flow-bin": "0.49.1",
+    "husky": "0.14.3",
+    "intl-messageformat": "2.0.0",
+    "intl-relativeformat": "2.0.0",
     "iron": "4.0.5",
     "jest": "20.0.4",
-    "lint-staged": "3.6.0",
-    "meetup-web-mocks": "1.0.190",
-    "mwp-cli": "1.0.71",
-    "prettier": "1.4.4",
-    "react": "15.5.4",
-    "react-addons-test-utils": "15.5.1",
+    "lint-staged": "4.0.1",
+    "meetup-web-mocks": "1.0.194",
+    "mwp-cli": "1.0.74",
+    "prettier": "1.5.2",
+    "react": "15.6.1",
+    "react-addons-test-utils": "15.6.0",
```

Comment:
this is an orphaned dependency and can be removed (it was moved into `react-dom` a month or so ago and import paths were updated